awsconfig.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import configparser
+import os
+from os.path import expanduser
+from optparse import OptionError, OptionParser, Option
```

Comment:
The import `optparse` is never used, please remove.

Reply:
@dancortes94 optparse it is used please  look at the main function, since Option, OptionError and OptionParser are being used.

Reply:
OptionError wasn't being used, OptionParser and Option were left intact.